docs/workflows/overview.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Overview
+sidebar_position: 5
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# How to integrate Beyond Identity into your application
+
+When thinking about how to integrate Beyond Identity into your application keep this in mind:
+ - Beyond Identity provides a number of user onboarding, passkey-generation, and passkey-authentication flows that can be integrated into your app. 
+ - There is not one-way to integrate these flows into your app. The user experience for every application is different, and as a developer you should use these flows however best fits your application’s user experience.
+ - Beyond Identity provides some example code and example applications, but don't take these examples as the only way to deploy.
+
+At a high level there are three key flows you'll need to understand to fully integrate Beyond Identity into your application.
+
+## User Provisioning
+
+Before Beyond Identity can authenticate a user it will need to know about them. Every user in the Beyond Identity system must be provisioned to a realm's directory in your tenant. There are three options to do this:
+ - **Provision Users via Admin Console** - You can add users through the Admin Console. This method is best for trying out Beyond Identity
+ - **Provision Users via API** - This is the most common way. Simply use the Beyond Identity identities API to create a users in your realm
+ - **Provision Users via SCIM** - SCIM is a protocol used by enterprise identity products to keep users, and their attributes in sync across SaaS products. If you already have a SSO such as Okta, Ping, Forgerock, Microsoft then you should consider using this method to provision your users.
```

Comment:
Should we link to these products? To Admin Console and to any Identities API or SCIM documentation?

Reply:
Yes good call will add